scripts/cluster/yarn_cluster_setup/move_files_to_master.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
+
+# current_path=`pwd`
+# library_file_address=$current_path/bash_scripts_library/include_functions.bash
+# source $library_file_address
+
+# get_nodes_info
+# authorize_access_between_nodes
+
+# Get the current directory
+base_address=`dirname "$(realpath $0)"`
+# THIS NEEDS TO CHANGE AS PER EXPERIMENT DETAILS
+master_node_login=pmrane@ms1028.utah.cloudlab.us
+# Project Folder name in root
+root_folder_in_server=/yarn_cluster_setup
+
+# Make Folder before moving files
+mkdir_command='sudo mkdir -p '${root_folder_in_server}
+ssh $master_node_login $mkdir_command
+
+# Move Files on Master Node's Home
+# Files Cannot be moved directly in Root due to permission issues
+scp -r ${base_address}/Test_Pi  $master_node_login:~/Test_Pi
```

Comment:
should not be in ~/

Reply:
I have replaced scp with rsync. That allowed me to move files and folders directly in root.